### PR TITLE
Support multiple editions

### PR DESCRIPTION
**Pull request by zoenolan**
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Does your submission pass tests?
* [x] Have you linted your code locally prior to submission?

### CODE REVIEW

**Comment by zoenolan** [comment]
This mostly fixes https://github.com/axna/zien/issues/72 apart from the Carousel. it would just to good to get some feedback  on this

File: src/components/Grid.js [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+		'editionid': 'ff-painful',
+		'name': 'Painful',
+		'artist': 'Fd Fornieles',
+		'year': '2019',
+		'type': 'Digital',
+		'thumbnail': ffpainfulartworkLink
+	}
+];
+
+var editionList = works.map(function(edition){
```
Comment (rehat101) [inline src/components/Grid.js]: Can you use 'const' instead of 'var' so we conform to ES6.

File: src/components/GridItem.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	}
+
+	h1 {
+		font-size: 1rem;
+	}
+`;
+
+function GridItem(item) {
+	return (
+			<Link to={"/editions/"+item.editionid}>
```
Comment (rehat101) [inline src/components/GridItem.js]: I believe you're not escaping strings properly here. We can utilize ES6 string literals.

for ex:
```javascript 
<Link to={`/editions/${item.edition.id}`}>
```

File: src/components/GridItem.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+				<GridItemStyle>
+					<ImgContainer>
+						<img alt={item.artist+", "+item.name+", "+item.year} src={item.thumbnail} />
```
Comment (rehat101) [inline src/components/GridItem.js]: same goes here as well

``````

File: src/components/Grid.js (hunk 1 above)
Comment (zoenolan) [reply src/components/Grid.js]: Updated

File: src/components/GridItem.js (hunk 2 above)
Comment (zoenolan) [reply src/components/GridItem.js]: Updated

File: src/components/GridItem.js (hunk 3 above)
Comment (zoenolan) [reply src/components/GridItem.js]: Updated

**Comment by zoenolan** [comment]
Carousel also pulling the data now

**Comment by rehat101** [comment]
Looks good. However, the grid items are being rendered in each row rather than columns. Is this an issue with you too?

**Comment by zoenolan** [comment]
Yes. I get the same

**Comment by zoenolan** [comment]
So I'm good to merge this. Then I can look at fixing the grid layout. We already have enough going on in this pull request?

**Comment by rehat101** [comment]
Yep, doing it now.

**Pull request merged**